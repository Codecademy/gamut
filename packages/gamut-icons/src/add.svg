<svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
    <g fill-rule="evenodd">
        <path d="M11 4h2v16h-2z"/>
        <path d="M20 11v2H4v-2z"/>
    </g>
</svg>
