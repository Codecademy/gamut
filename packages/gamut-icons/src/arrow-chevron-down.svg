<svg fill="currentColor" version="1.1" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Chevron Down Icon</title><path d="M20.293 7.293l1.414 1.414L12 18.414 2.293 8.707l1.414-1.414L12 15.586z"></path></svg>