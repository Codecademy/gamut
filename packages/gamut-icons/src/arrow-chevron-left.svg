<svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
    <path d="M9.475 12.06l7.293 7.294-1.414 1.414-8.708-8.707 8.708-8.707 1.414 1.414z"/>
</svg>
