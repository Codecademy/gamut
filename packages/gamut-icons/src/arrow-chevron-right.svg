<svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
    <path d="M13.94 12.06L6.645 4.769 8.06 3.355l8.707 8.707-8.707 8.707-1.415-1.414z"/>
</svg>
