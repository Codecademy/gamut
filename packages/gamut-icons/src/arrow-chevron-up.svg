<svg fill="currentColor" version="1.1" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Chevron Up Icon</title><path d="M20.293 18.414L21.707 17 12 7.293 2.293 17l1.414 1.414L12 10.121z"></path></svg>