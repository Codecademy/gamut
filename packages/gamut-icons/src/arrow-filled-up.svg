<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
    <path fill="#444" d="M8 19l8-6 8 6z"/>
</svg>
