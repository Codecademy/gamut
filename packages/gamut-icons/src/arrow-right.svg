<svg fill="currentColor" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
    <path d="M4 10.92v2h12l-5.5 5.5 1.42 1.42 7.92-7.92L11.92 4 10.5 5.42l5.5 5.5z"/>
</svg>
