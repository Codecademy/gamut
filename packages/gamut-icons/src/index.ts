export * from './icons/regular';
export * from './icons/mini';
export * from './types';
