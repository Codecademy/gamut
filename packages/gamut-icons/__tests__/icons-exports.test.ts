import * as icons from '../src/index';

test('Icons Exported Keys 1', () => {
  const exportedNames = Object.keys(icons).sort((a, b) =>
    a.toLowerCase().localeCompare(b.toLowerCase())
  );
  expect(exportedNames).toMatchSnapshot();
});
