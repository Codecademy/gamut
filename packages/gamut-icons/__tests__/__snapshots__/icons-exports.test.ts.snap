// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons Exported Keys 1 1`] = `
[
  "AccountingCoinsIcon",
  "AddBoldIcon",
  "AddIcon",
  "AiChatSparkFilledIcon",
  "AiChatSparkIcon",
  "AiEditSparkIcon",
  "AlarmClockIcon",
  "AlertFilledIcon",
  "AlertIcon",
  "AmexIcon",
  "AnalyticsGraphIcon",
  "AppleIcon",
  "ArchiveIcon",
  "ArrowChevronDownFilledIcon",
  "ArrowChevronDownIcon",
  "ArrowChevronLeftIcon",
  "ArrowChevronRightBoldIcon",
  "ArrowChevronRightIcon",
  "ArrowChevronUpIcon",
  "ArrowDashRightIcon",
  "ArrowDownIcon",
  "ArrowLeftIcon",
  "ArrowRightIcon",
  "ArrowThickCircleRightIcon",
  "ArrowThickCircleUpIcon",
  "ArrowUpIcon",
  "ArticleIcon",
  "ArtificialIntelligenceIcon",
  "AssessmentsIcon",
  "BadgeIcon",
  "BashShellIcon",
  "BellIcon",
  "BookBookmarkIcon",
  "BookClosedBookmarkIcon",
  "BookFlipPageIcon",
  "BookLibraryIcon",
  "BriefcaseIcon",
  "BrowserPageLayoutIcon",
  "BugIcon",
  "BulbIcon",
  "CalendarIcon",
  "CallIcon",
  "CertificateIcon",
  "ChallengeProjectIcon",
  "ChatIcon",
  "CheckCircledIcon",
  "CheckFilledIcon",
  "CheckHeavyIcon",
  "CheckIcon",
  "ChecklistIcon",
  "CIcon",
  "ClockIcon",
  "CloseCircleIcon",
  "ClosedCaptionDisabledIcon",
  "ClosedCaptionIcon",
  "CloseIcon",
  "CloseSquareIcon",
  "CloudCheckIcon",
  "CloudComputingIcon",
  "CloudIcon",
  "ClubsSealIcon",
  "CommentsIcon",
  "CommonFileSettingsIcon",
  "CommonFileTextHeartIcon",
  "CommonFileTextQuestionIcon",
  "CommunityIcon",
  "ComputerScienceIcon",
  "ConversationChatIcon",
  "CopyIcon",
  "CPlusIcon",
  "CreditCard1Icon",
  "CreditCardIcon",
  "CSharpIcon",
  "CubeIcon",
  "CybersecurityIcon",
  "DataAnalyticsIcon",
  "DataEngineeringIcon",
  "DataFileBarsShareIcon",
  "DataScienceIcon",
  "DataTransferVerticalIcon",
  "DataVisualizationIcon",
  "DeleteFilledIcon",
  "DeleteIcon",
  "DevopsIcon",
  "DiamondIcon",
  "DiscordIcon",
  "DiscordOutlineIcon",
  "DiscountStarPremiumFilledIcon",
  "DownloadIcon",
  "DuplicateIcon",
  "EaIcon",
  "EarthIcon",
  "EditIcon",
  "EditorIcon",
  "EmailFilledIcon",
  "EmailIcon",
  "ExperienceIcon",
  "ExploreIcon",
  "FacebookIcon",
  "FallbackSkillIcon",
  "FaviconIcon",
  "FaviconSolidIcon",
  "FileAddIcon",
  "FileCodeAdd1Icon",
  "FileDownloadIcon",
  "FileEditIcon",
  "FileIcon",
  "FileLockIcon",
  "FilePyIcon",
  "FileSearchIcon",
  "FileTextInfoIcon",
  "FilterIcon",
  "FitnessDumbbellIcon",
  "FitNormalIcon",
  "FitShrinkIcon",
  "FlutterIcon",
  "FolderAddIcon",
  "FolderIcon",
  "ForkIcon",
  "ForumsChatIcon",
  "FullscreenIcon",
  "FullscreenSquareIcon",
  "GameDevelopmentIcon",
  "GaugeDashboardIcon",
  "GearIcon",
  "GithubIcon",
  "GithubOutlineIcon",
  "GitIcon",
  "GlassesIcon",
  "GoIcon",
  "GolfHoleBallIcon",
  "GoogleIcon",
  "GplusIcon",
  "GraphStatsAscendIcon",
  "GrowthIcon",
  "HalloweenBroomIcon",
  "HammerWrenchIcon",
  "HouseEntranceIcon",
  "HtmlCssIcon",
  "HyperlinkIcon",
  "IbmIcon",
  "ImageIcon",
  "InfoCircleIcon",
  "InformationalIcon",
  "InProgressIcon",
  "InstagramIcon",
  "InstructionsIcon",
  "InteractiveCursorIcon",
  "IntersectionIcon",
  "ItIcon",
  "JavaIcon",
  "JavascriptIcon",
  "JiraIcon",
  "KeyboardIcon",
  "KotlinIcon",
  "LayoutModuleAltIcon",
  "LayoutModuleIcon",
  "LearnIcon",
  "LessonIcon",
  "LevelIcon",
  "LifePreserverIcon",
  "LinkedinIcon",
  "LinkedinOutlineIcon",
  "LinkIcon",
  "ListeningIcon",
  "ListIcon",
  "LiveVideoIcon",
  "LocationPinIcon",
  "LockHeavyIcon",
  "LockIcon",
  "LogoutIcon",
  "MachineLearningIcon",
  "MaintenanceIcon",
  "MapFingerIcon",
  "MapPinIcon",
  "MarkdownIcon",
  "MarkerIcon",
  "MastercardIcon",
  "MathIcon",
  "MediumIcon",
  "MegaphoneIcon",
  "MenuIcon",
  "MetaIcon",
  "MicrophoneIcon",
  "MicrosoftIcon",
  "MiniAccessibilityIcon",
  "MiniAddIcon",
  "MiniArrowDownIcon",
  "MiniArrowLeftIcon",
  "MiniArrowRightIcon",
  "MiniArrowUpIcon",
  "MiniAudioDescriptionIcon",
  "MiniCheckCircleIcon",
  "MiniChevronDownIcon",
  "MiniChevronLeftIcon",
  "MiniChevronRightIcon",
  "MiniChevronUpIcon",
  "MiniCircleIcon",
  "MiniCloseCaptioningIcon",
  "MiniCopyIcon",
  "MiniCursorIcon",
  "MiniDeleteIcon",
  "MiniEditIcon",
  "MiniInfoCircleIcon",
  "MiniInfoOutlineIcon",
  "MiniKebabMenuIcon",
  "MiniLiveVideoIcon",
  "MinimizeBarIcon",
  "MinimizeIcon",
  "MiniOpenIcon",
  "MiniPinIcon",
  "MiniQuoteIcon",
  "MiniRemoveCircleIcon",
  "MiniReorderDotsVerticalIcon",
  "MiniRibbonIcon",
  "MiniStarIcon",
  "MiniThumbsDownIcon",
  "MiniThumbsUpIcon",
  "MiniUndoIcon",
  "MiniWarningTriangleIcon",
  "MobileDevelopmentIcon",
  "MoneyBackGuaranteeIcon",
  "MoneyWalletIcon",
  "MonitorBookIcon",
  "MoveLeftIcon",
  "MoveRightIcon",
  "MultipleActionsViewIcon",
  "MultipleUsersIcon",
  "NarrativeIcon",
  "NetworkUserIcon",
  "NotebookIcon",
  "ObjectiveIcon",
  "OfficeBuildingDoubleIcon",
  "OneTimeIcon",
  "OnlineClassStudentIcon",
  "OpenAiFullIcon",
  "OpenAiIcon",
  "OpenIcon",
  "OptimizationClockIcon",
  "OverviewIcon",
  "PaginateFilterTextIcon",
  "ParagraphJustifiedAlignIcon",
  "PathChangeIcon",
  "PathCompleteIcon",
  "PathIcon",
  "PauseIcon",
  "PaypalLogoIcon",
  "PdfIcon",
  "PeopleIcon",
  "PercipioIcon",
  "PersonChatIcon",
  "PersonFilledIcon",
  "PersonIcon",
  "PersonPonytailIcon",
  "PhoneIcon",
  "PhpIcon",
  "PieLineGraphIcon",
  "PinIcon",
  "PipExitIcon",
  "PipIcon",
  "PlayCircledIcon",
  "PlayIcon",
  "PortfolioProjectIcon",
  "ProfessionalCertificateIcon",
  "ProgrammingBrowserIcon",
  "ProgrammingTeamChatIcon",
  "ProjectsIcon",
  "PythonIcon",
  "QrCodeScanIcon",
  "QuestionMarkIcon",
  "QuizIcon",
  "RadarIcon",
  "RailsIcon",
  "RankingWinnerIcon",
  "RatingStarCircleIcon",
  "RatingStarGiveIcon",
  "ReactIcon",
  "RecordingIcon",
  "RedditIcon",
  "RedditSolidIcon",
  "RedoIcon",
  "RefreshIcon",
  "ReportIcon",
  "ResetIcon",
  "ResizableHandleIcon",
  "ResponsiveIcon",
  "RibbonDoubleFilledIcon",
  "RibbonDoubleIcon",
  "RibbonFilledIcon",
  "RibbonIcon",
  "RIcon",
  "RoadmapIcon",
  "RocketIcon",
  "RoundPinIcon",
  "RubyIcon",
  "RulerTriangleIcon",
  "SavingBankIcon",
  "SearchIcon",
  "SendIcon",
  "SettingsSliderVerticalIcon",
  "ShareAltIcon",
  "ShareIcon",
  "SlackIcon",
  "SmallCheckIcon",
  "SmileyHappyIcon",
  "SmileyIndifferentIcon",
  "SmileySadIcon",
  "SmileyStarEyesIcon",
  "SparkleIcon",
  "SparkleOutlineIcon",
  "SpotifyIcon",
  "SqlIcon",
  "StackedSquaresIcon",
  "StarCircleIcon",
  "StarFilledCircleIcon",
  "StarFilledIcon",
  "StarIcon",
  "StopSignIcon",
  "StreakIcon",
  "StudyBookIcon",
  "SupportFilledIcon",
  "SupportIcon",
  "SwiftIcon",
  "SymbolEqualIcon",
  "SynchronizeRefreshArrowIcon",
  "TagIcon",
  "TaskListMultipleIcon",
  "TerminalIcon",
  "ThumbsDownFilledIcon",
  "ThumbsDownIcon",
  "ThumbsUpFilledIcon",
  "ThumbsUpIcon",
  "TikTokIcon",
  "Timer10Icon",
  "TrackIcon",
  "TranslateIcon",
  "TrashIcon",
  "TrophyFilledIcon",
  "TrophyIcon",
  "TwitterIcon",
  "TwitterOutlineIcon",
  "TypeIcon",
  "TypescriptIcon",
  "UndoIcon",
  "UnlockHeavyIcon",
  "UnlockIcon",
  "UploadIcon",
  "VideoPlayerMovieIcon",
  "ViewIcon",
  "ViewOffIcon",
  "VisaIcon",
  "VolumeControlFullIcon",
  "VolumeControlMediumIcon",
  "VolumeControlMuteIcon",
  "WebDesignIcon",
  "WebDevelopmentIcon",
  "WebFormProgressIcon",
  "WhatsAppFilledIcon",
  "XIcon",
  "YoutubeIcon",
  "YoutubePlayIcon",
  "ZoomIcon",
]
`;
